internal/tfengine/tfengine_test.go

```diff
@@ -0,0 +1,35 @@
+package tfengine
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+const basePath = "../../examples/tfengine"
+
+// TestExamples is a basic test for the engine to ensure it runs without error on the examples.
+func TestExamples(t *testing.T) {
+	exs := []string{"simple", "full"}
+	for _, ex := range exs {
+		t.Run(ex, func(t *testing.T) {
```

Comment:
Since you're using `t.Run()`, why not try all examples in parallel?

Reply:
I ran a comparison and it doesn't seem to save any time. Given this is a very quick test there isn't much benefit from parallelization. In this case I would prefer the option with fewer lines of code.